src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public abstract class Figure implements Picture {
```

Comment:
Don't use abstract classes to set behavior for classes. In this case, you are using an abstract class to define a common property for all figures, which is fine, but make sure you are not using it to define behaviors that should be implemented differently in each subclass. If you need to enforce a contract for methods like `calculateArea()`, consider using an interface instead.